lib/plugins/alexa.js

```diff
@@ -0,0 +1,110 @@
+var _ = require('lodash');
+var async = require('async');
+
+function init (env, ctx) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['ctx' is defined but never used. (no-unused-vars)](https://app.codacy.com/manual/m-kozak/cgm-remote-monitor/pullRequest?prid=1196151)